integration-tests/utils/integration-helpers.js

```diff
@@ -122,18 +131,52 @@ const deleteWorkspace = withSignedInPage(async ({ page, billingProject, workspac
   }
 });
 
-const withWorkspace = (test) => async (options) => {
-  console.log('withWorkspace ...');
-  const workspaceName = await makeWorkspace(options);
+const deleteWorkspaceV2 = withSignedInPage(async ({ page, billingProject, workspaceName }) => {
```

Comment:
Maybe a comment to say that this should not be used with our current (static workspace) testing strategy?

I'm imagining a scenario where someone decides that they need an azure e2e test, includes this for cleanup, and thus accidentally deletes our static infrastructure.

Reply:
I don't think there is any static workspace infrastructure yet.

Reply:
We aren't using a static workspace currently